hawkbit-extension-multi-user/README.md

```diff
@@ -0,0 +1,25 @@
+# hawkBit multi user extension
+
+This extension allows you to statically configure multiple users through the application properties.
+When the extension is enabled but no users are configured, it will fall back to the standard security.user configuration.
+
+## Example configuration
+
+A list of permissions with description is [available from the hawkBit repository](https://github.com/eclipse/hawkbit/blob/master/hawkbit-security-core/src/main/java/org/eclipse/hawkbit/im/authentication/SpPermission.java).
+Additionally, if a single permission with the name *ALL* is configured, that user will be granted all available permissions.
+
+    multiuser.user[0].username=admin
+    multiuser.user[0].password=admin
+    multiuser.user[0].firstname=Test
+    multiuser.user[0].lastname=Admin
+    multiuser.user[0].email=admin@test.de
+    multiuser.user[0].permissions[0]=ALL
+    multiuser.user[1].username=test
+    multiuser.user[1].password=test
+    multiuser.user[1].firstname=Test
+    multiuser.user[1].lastname=Tester
+    multiuser.user[1].email=test@tester.com
+    multiuser.user[1].permissions[0]=READ_TARGET
```

Comment:
I'd prefer a flat list like this:
```suggestion
    multiuser.user[1].permissions=READ_TARGET,UPDATE_TARGET,CREATE_TARGET,DELETE_TARGET
```
Otherwise if you'll add more permissions for multiple users (let's say all except one or two) the property file is getting longer and longer. Spring will take care of the mapping so your current implementation won't brake by this change.